### DISCUSSION TITLE
Initial feedback

### DISCUSSION THREAD

**Discussion by collinjc** [rhinstaller/anaconda-webui#38] [General]
Overall, I like where the Anaconda Web UI seems to be heading. Visually, it feels clean, fresh, and modern. I'm aware that certain functionality (disk partitioning, package selection, etc.) hasn't been implemented yet, so I won't touch on those items.

One thing that was mentioned [a while back](https://fedoraproject.org/wiki/Changes/Anaconda_Web_UI_preview_image#Detailed_Description) was that the Web UI would allow for "local and remote access to the Web UI." Is remote access functional at this stage? When hitting port 9090, however, an error stating that "Server is missing the cockpit-system package," is all that is seen.

Next, I feel that the ability to close the installer window via the 'x' button at the top right should probably be removed. Clicking this causes the installer to immediately halt and kill the installation, leaving the system in an unknown state. While the user should certainly be able to cancel the installation, I feel that they should probably do so by clicking 'Quit' in the Web UI so that they can be prompted before the installer exits.

Finally, prior to the final release, some or all of the right-click menu items should probably be disabled -- especially 'Inspect Element.' This is currently handy for debugging, but if accidentally triggered by the user, it is very easy to cause undesired behavior (such as being logged out of cockpit with no obvious way to get back to the installation process).

Overall, I'm excited at the direction this is headed, and look forward to seeing future development.

**Comment by micl2e2** [comment]
Same experience if accessing http://x.x.x.x:9090: Got "Server is missing the cockpit-system package".

Try access 
```
http://<your-ip-addr>:9090/cockpit/@localhost/anaconda-webui/index.html
```
instead, it seems like an undocumented URL for accessing Anaconda WebUI's interface, so its path might change in the future.

It'd be better this URL appear directly on, like the first step's UI, so that the users are informed that they can access the installer remotely, but currently it is not true. 

Does Anaconda Team plans to make it more accessible in the future releases ? @jkonecny12

**Reply by jkonecny12** [reply]
Hi, yes we would like to make it better to access but right now it's not part of the testing / feedback. We still need to resolve some issues there.
